ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -165,6 +165,11 @@ public void updateExtraData(ReactEditText view, Object extraData) {
     }
   }
 
+  @ReactProp(name = ViewProps.COLOR, defaultInt = 0, customType="color")
```

Comment:
Can you remove the defaultInt and check one thing for me?  If you set the color to something and then you rerender with no color set, does it revert back to the proper color?